internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,15 +132,17 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
-	files, err := filepath.Glob(filepath.Join(definitionsDir, "*.yaml"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func findKubernetesDefinitions(definitionsDir string) (string, error) {
+	if _, err := os.Stat(definitionsDir); err != nil {
+		return "", errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
```

Comment:
`definitions directory` is not consistent with `definitionPath` any longer. I think we should either use the plural in every case (eg. definitionsPath) or come up with a better name